content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+The second critique concerns what @thompson_mind_2007 has termed the "informational dualism" of the notion of representation in cognitive science:
+at the concrete level, it questions whether a representationalist approach can naturalise meaning, a core aspect in the analysis of culture and life;
+at the programmatic level it suggests that, compared to the focus on dynamic couplings that Thompson and his colleagues develop under the "enactive approach" banner, a focus on representations in cognitive science has less heuristic value in guiding the exploration of the interactions between culture and psychology.
+Regardless of whether proponents can reconcile SCE and CAT with the second critique, I believe both approaches can benefit greatly from these debates, and that empirical study has a crucial role to play in exploring the implications of each side.
+
+The empirical exploration of CAT has proven a difficult task, for two main reasons.
+First, Sperber clarifies that CAT is not, and should not be, a "grand unitary theory":
+
+> "An epidemiological approach ... should not hope for one grand unitary theory. It should, rather, try to provide interesting questions and useful conceptual tools, and to develop the different models needed to explain the existence and fate of the various families of cultural representations." [@sperber_explaining_1996, p. 83]
+
+Indeed, CAT accomplishes this:
+
+> "What the epidemiological analogy suggests is a general approach, types of questions to ask, ways of constructing concepts, and a plurality of not too grand theoretical aims." [@sperber_explaining_1996, p. 61]
+
+As a consequence, testing CAT means evaluating the fruitfulness of its paradigm.
+No single study, or collection of studies for that matter, will reach a yes or no answer to the validity of CAT.
+But we can reach a collective consensus about the usefulness of approaching the psychology-culture link with the tools and questions developed by that theory.
+
+As the name CAT indicates, an important part of that toolbox is the notion of cultural attractor, which is amenable to empirical study:
+the presence or absence of a well-defined attractor in a given situation can be turned into a testable hypothesis.
+In addition at the meta-analytical level, one can readily evaluate the usefulness of a cultural attraction approach in concrete domains, by analysing whether or not the questions it encourages result in fruitful studies, on average, for that domain.
+So while testing CAT empirically is not a simple matter of yes or no, the theory provides a clear set of tools and lines of thought that are well worth evaluating in a wider, programmatic appraisal.
+
+The second challenge to empirical exploration resides in the fact that quantitative data on out-of-laboratory cultural artefacts is not easy to collect.
```

Comment:
Pour la description plus précise, tu fais référence aux [changements](https://github.com/wehlutyk/thesis/pull/3#discussion_r124531273) [ci-dessus](https://github.com/wehlutyk/thesis/pull/3#discussion_r124530112) ?

TODO
- [ ] spell out the correspondence of attractor/public-/private-representation with these examples